core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/DocFormatter.java

```diff
@@ -2,16 +2,66 @@
 
 import java.io.IOException;
 import java.io.Writer;
+import java.text.Normalizer;
 import java.util.List;
 
-import io.quarkus.annotation.processor.Constants;
-
 interface DocFormatter {
-    default String getAnchor(String key) {
-        return key.replaceAll("[<\">]", Constants.EMPTY);
+    default String getAnchor(String string) {
+        // remove accents
+        string = Normalizer.normalize(string, Normalizer.Form.NFKC)
+                .replaceAll("[àáâãäåāąă]", "a")
+                .replaceAll("[çćčĉċ]", "c")
+                .replaceAll("[ďđð]", "d")
+                .replaceAll("[èéêëēęěĕė]", "e")
+                .replaceAll("[ƒſ]", "f")
+                .replaceAll("[ĝğġģ]", "g")
+                .replaceAll("[ĥħ]", "h")
+                .replaceAll("[ìíîïīĩĭįı]", "i")
+                .replaceAll("[ĳĵ]", "j")
+                .replaceAll("[ķĸ]", "k")
+                .replaceAll("[łľĺļŀ]", "l")
+                .replaceAll("[ñńňņŉŋ]", "n")
+                .replaceAll("[òóôõöøōőŏœ]", "o")
+                .replaceAll("[Þþ]", "p")
+                .replaceAll("[ŕřŗ]", "r")
+                .replaceAll("[śšşŝș]", "s")
+                .replaceAll("[ťţŧț]", "t")
+                .replaceAll("[ùúûüūůűŭũų]", "u")
+                .replaceAll("[ŵ]", "w")
+                .replaceAll("[ýÿŷ]", "y")
+                .replaceAll("[žżź]", "z")
+                .replaceAll("[æ]", "ae")
+                .replaceAll("[ÀÁÂÃÄÅĀĄĂ]", "A")
+                .replaceAll("[ÇĆČĈĊ]", "C")
+                .replaceAll("[ĎĐÐ]", "D")
+                .replaceAll("[ÈÉÊËĒĘĚĔĖ]", "E")
+                .replaceAll("[ĜĞĠĢ]", "G")
+                .replaceAll("[ĤĦ]", "H")
+                .replaceAll("[ÌÍÎÏĪĨĬĮİ]", "I")
+                .replaceAll("[Ĵ]", "J")
+                .replaceAll("[Ķ]", "K")
+                .replaceAll("[ŁĽĹĻĿ]", "L")
+                .replaceAll("[ÑŃŇŅŊ]", "N")
+                .replaceAll("[ÒÓÔÕÖØŌŐŎ]", "O")
+                .replaceAll("[ŔŘŖ]", "R")
+                .replaceAll("[ŚŠŞŜȘ]", "S")
+                .replaceAll("[ÙÚÛÜŪŮŰŬŨŲ]", "U")
+                .replaceAll("[Ŵ]", "W")
+                .replaceAll("[ÝŶŸ]", "Y")
+                .replaceAll("[ŹŽŻ]", "Z")
+                .replaceAll("[ß]", "ss");
```

Comment:
Very... thorough :)

Reply:
Yeah, I took this from a French source with years of experience dealing with I18n ;)